operations/team-compass.md

```diff
@@ -17,10 +17,26 @@ There are a few major reasons why we use a Team Compass:
 
 ## Update the team compass
```

Comment:
Suggest we make this "How to update the team compass" to make it clearer (thinking w/ my Diataxis hat on). Sorry I can't propose the change because GH isn't letting me

Reply:
Agreed.